### ISSUE TITLE
openxml4net temp files

### ISSUE THREAD

**Issue by Hauptsatana** [dotnetcore/NPOI#32]
I've recently migrated to .NET Core 2.0 from .NET Framework 4.6.2. So instead of NPOI 2.3.0 I have to use DotNetCore.NPOI. When NPOI reads workbook it creates *.tmp file in working directory and doesn't delete it after workbook is closed. Is it a bug or some additional configuration must be added to delete these files?

**Comment by yuleyule66** [comment]
@baiyunchen 
Hi, yunchen
  I'm on a business trip recently. Can you see what the problem is?

**Comment by baiyunchen** [comment]
@Hauptsatana  Can you Provide you sample code? 
I tried to reproduce what you said with the following code and did not found the .tmp file generated.And I showd windows hidden files and system files.
```
public void ReadExcel()
{
    using (var file = new FileStream(@"template.xlsx", FileMode.Open, FileAccess.Read))
    {
        var excel = new XSSFWorkbook(file);
        var cell = excel.GetSheetAt(0).GetRow(0).GetCell(0);
        Console.WriteLine(cell.NumericCellValue);
    }

    using (var file = new FileStream(@"template.xls", FileMode.Open, FileAccess.Read))
    {
        var excel = new HSSFWorkbook(file);
        var cell = excel.GetSheetAt(0).GetRow(1).GetCell(2);
        Console.WriteLine(cell.NumericCellValue);
    }
}
```

**Comment by Hauptsatana** [comment]
@baiyunchen here is code that I used
`string filePath = "template.xlsx";IWorkbook workbook = WorkbookFactory.Create(filePath);`
...
workbook.Close()` 

Tests have shown that .tmp file is produced after `workbook.Close()` when workbook was opened using string filePath.
If I use FileStream as in your example or exclude `workbook.Close()` then no .tmp file is generated.

**Comment by baiyunchen** [comment]
@Hauptsatana  It's a bug.
Please reference https://github.com/dotnetcore/NPOI/pull/35.

**Comment by yuleyule66** [comment]
Fixed in version 1.2.0
